integreat_cms/core/management/commands/update_link_text.py

```diff
@@ -0,0 +1,138 @@
+    Management command to update the link text of ALL links with the given URL.
+    ALL links receive the SAME new link text.
+    This applies in ALL regions.
+    """
+
+    help = "Update link text of the URL"
+
+    def add_arguments(self, parser: CommandParser) -> None:
+        """
+        Define the arguments of this command
+
+        :param parser: The argument parser
+        """
+        parser.add_argument(
+            "--target-url", help="The URL whose link text needs to be changed"
+        )
+        parser.add_argument("--new-link-text", help="The new link text")
+        parser.add_argument("--username", help="The username of the creator")
+
+    def handle(
+        self,
+        *args: Any,
+        target_url: str,
+        new_link_text: str,
+        username: str,
+        **options: Any,
+    ) -> None:
+        r"""
+        Try to run the command
+
+        :param \*args: The supplied arguments
+        :param target_url: URL whose link texts need to be updated
+        :param new_link_text: New link text to replace with the current link texts
+        :param username: The username of the creator
+        :param \**options: The supplied keyword options
+        :raises ~django.core.management.base.CommandError: When the input is invalid
+        """
+        self.set_logging_stream()
+
+        if not target_url:
+            raise CommandError("Please specify the target URL.")
+
+        if not new_link_text:
+            raise CommandError("Please specify the new link text.")
+
+        target_url_object = get_url(target_url)
+        user = get_user(username) if username else None
+
+        target_links = target_url_object.links.all()
+
+        target_contents = {target_link.content_object for target_link in target_links}
+
+        for target_content in target_contents:
+            if target_content._meta.default_related_name in [
+                "page_translations",
+                "event_translations",
+                "poi_translations",
+                "imprint_translations",
+            ]:
+                try:
+                    content = fromstring(target_content.content)
+                except ParserError:
+                    logger.info(
+                        "A link text in %r could not be updated automatically.",
+                        target_content,
+                    )
+
+                for elem, _, link, _ in content.iterlinks():
+                    if link == target_url:
+                        elem.text = new_link_text
+
+                new_translation = target_content.create_new_version_copy(user)
+                new_translation.content = unescape(
+                    tostring(content, encoding="unicode", with_tail=False)
+                )
+
+                if new_translation.content != target_content.content:
+                    target_content.links.all().delete()
+                    new_translation.save()
+        time.sleep(0.1)
+        tasks_queue.join()
```

Comment:
Same with this join. I think I know what it does, but I don't think it is immediately obvious, especially if the reader only has superficial knowledge about `linkcheck`
```suggestion
        # Wait until all background tasks of linkcheck indexing the new content objects have been processed
        tasks_queue.join()
```